simple/cq_data.c

```diff
@@ -481,7 +481,7 @@ int main(int argc, char **argv)
 
 	hints->ep_attr->type = FI_EP_MSG;
 	hints->caps = FI_MSG;
-	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR;
+	hints->mode = FI_LOCAL_MR | FI_PROV_MR_ATTR | FI_RX_CQ_DATA;
```

Comment:
Got it! I shall update the verbs provider to check for this mode bit when user requests FI_RMA in capabilities or if the cq_data_size domain attribute is set. I shall remove the mode bit from this test and update msg_rma to have it.